### ISSUE TITLE
Make switcher to "Unread only" mode in left panel room list

### ISSUE THREAD

**Issue by MurzNN** [element-hq/element-web#16009]
**Is your suggestion related to a problem? Please describe.**
Most of times users look at rooms list to see unread rooms, that they must read. Rare they search read rooms for read them again, or post a new message. So long list of all joined rooms is needed not so often.

**Describe the solution you'd like**
For make easier to find all unread rooms, especially when we use tags for grouping rooms (that generate many groups in list), will be good to add "Unread only" switcher in left panel room list - when it "on" - list will filter only unread rooms, if "off" - all rooms, like now. 

On Element-Android this function is performed by "Notifications" tab and it is very useful!

Implementation of that switcher on desktop you can see in action, for example, in https://vk.com/messenger (at bottom).

**Comment by SimonBrandner** [comment]
Currently, there is a button labelled `Notifications` in the top right corner that should do something like the Android Notifications tab, as I understand it. The problem is that it sometimes acts very weirdly. 

The Element core team seems to be aware of that since in the [Element Public Roadmap](https://github.com/vector-im/roadmap/projects/1) you can find [Notifications](https://github.com/vector-im/roadmap/issues/6). Currently, they are working on improving VoIP and Spaces. From the current roadmap, it seems that improving Notifications should be worked on after that.

**Comment by MurzNN** [comment]
> @MurzNN Currently, there is a button labelled Notifications in the top right corner that should do something like the Android Notifications tab, as I understand it. The problem is that it sometimes acts very weirdly.

No, current "Notifications" button do the other thing: show list of last messages, targeted to me directly or via mentions. But we need the way to quickly filter-out the whole list of rooms by "Unread" status.

About Roadmap - no problem, I create this issue as suggestion for future implementations.

**Comment by SimonBrandner** [comment]
> No, current "Notifications" button do the other thing: show list of last messages, targeted to me directly or via mentions. But we need the way to quickly filter-out the whole list of rooms by "Unread" status, instead of list with last messages.

@MurzNN Thank you for the correction!

**Comment by yousefamar** [comment]
Just to add to this, another good alternative would be to be able to use the filter bar with something like `is:unread`, the same way many chat clients have DSL for searching. That, or to be able to sort the left panel by unread (even in a similar way to how there is "low priority" currently)

**Comment by t3chguy** [comment]
You *can* sort by unread:

Each sublist has its own `...`
![image](https://user-images.githubusercontent.com/2403652/103140510-3c678280-46df-11eb-94fa-32b3bd91cb15.png)

**Comment by yousefamar** [comment]
Oh, thanks, I didn't find that, sorry!

**Comment by andybalaam** [comment]
Related: [#7185](https://github.com/vector-im/element-web/issues/7185)

**Comment by atomdmac** [comment]
This remains my most-hoped-for feature in Element.  I use `CMD-K` exclusively to navigate between rooms on apps like Slack and Discord and having unread rooms sorted to the top makes that so much easier.

As someone with no familiarity with the inner workings of Element, it's odd to me that this hasn't been implemented for the switcher since (as @t3chguy points out) effectively identical functionality exists for lists in the sidebar (though it is less keyboard-accessible).

**Comment by t3chguy** [comment]
Fixed by new room list

**Comment by andybalaam** [comment]
Related: https://github.com/element-hq/element-web/issues/30910
